src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { CommentListType } from '../../type/comment';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+export const CommentList: React.FC<CommentListType> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo comment={comment} />
```

Comment:
Add a key prop to CommentInfo component to avoid potential issues with rendering and performance.